Update parameters in runtime add asset and add app (#539)
scale-it/algo-builder: CHANGELOG.md

    const appID = receipt.appID;
    ```
+ `getProgram` is moved to `@algo-builder/runtime` from `@algo-builder/algob`.
+ Renamed `Runtime.addApp` to `Runtime.deployApp`, `Runtime.addAsset` to `Runtime.deployAsset`, `Runtime.addAssetDef` to `Runtime.deployAssetDef`. 

> + Make order of `Runtime.deloyApp` like `Deloyer.deployApp`.
>     * For instance:

Reviewer: ```suggestion
+ Update `Runtime.deloyApp` to be compatible with `Deloyer.deployApp`. Example:
```

    ```javascript
    //from 
    Runtime.deployApp(
        flags: AppDeploymentFlags, 
        payFlags: types.TxParams,
        approvalProgram: string, 
        clearProgram: string,
        debugStack?: number
    )
    //to
    Runtime.deployApp(
        approvalProgram: string, 
        clearProgram: string,
        flags: AppDeploymentFlags, 
        payFlags: types.TxParams,
        debugStack?: number
    )

>     ```

Reviewer: please add a changelog entry that `runtime.addAsset` ... are deprecated

    const appID = receipt.appID;
    ```
+ `getProgram` is moved to `@algo-builder/runtime` from `@algo-builder/algob`.
+ `runtime.addAsset`, `runtime.addAssetDef` and `runtime.addApp` are deprecated. 

> Please use `runtime.deployASA`, `runtime.deployASADef` and `runtime.deployAdd` instead of the above functions.

Reviewer 2: ```suggestion
Please use `runtime.deployASA`, `runtime.deployASADef` and `runtime.deployApp` instead of the above functions.
```
```suggestion
Please use `runtime.deployASA`, `runtime.deployASADef` and `runtime.deployAdd` instead of the above functions.
```

+ Update `runtime.deloyApp` to be compatible with `deployer.deployApp`. Example:
    * For instance:
    ```javascript
    // from 
    runtime.deployApp(
        flags: AppDeploymentFlags, 
        payFlags: types.TxParams,
        approvalProgram: string, 
        clearProgram: string,
        debugStack?: number
    )
    // to
    runtime.deployApp(
        approvalProgram: string, 
        clearProgram: string,
        flags: AppDeploymentFlags, 
        payFlags: types.TxParams,
        debugStack?: number

>     )

Reviewer 2: both functions are identical here
